api/DOMTokenList.json

```diff
@@ -61,10 +61,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "3.6"
```

Comment:
The parent api.DOMTokenList isn't updated in this PR, and has Firefox versions 3/4. That's surprising given that http://mdn-bcd-collector.appspot.com/tests/api/DOMTokenList is just looking at the interface object. Was there an interface object exposed but with no members in Firefox 3, and the members were added in 3.6? I'm guessing that actually the whole thing was added in Firefox 3.6, but then can't the collector figure that out from the test results?

Reply:
Yeah, this interface was actually added in Firefox 3.6.  I had a little VM misconfiguration where my Firefox 3 VM was actually 3.6.  I've since then fixed it, but apparently I had a few PRs with this data.

I can follow-up with a corrections PR, or include the correction for this interface in this PR, whichever is more preferred?

Reply:
If you could fix all of DOMTokenList in this PR that would be great. Other 3 vs. 3.6 mishaps as its own PR would be good.